core/src/main/java/io/aiven/klaw/service/UsersTeamsControllerService.java

```diff
@@ -588,7 +588,12 @@ public ApiResponse addNewUser(UserInfoModel newUser, boolean isExternal) throws
       userInfo.setPwd(newUser.getUserPassword());
       String result = dbHandle.addNewUser(userInfo);
 
+      if (ApiResultStatus.SUCCESS.value.equals(result)) {
+        manageDatabase.addUserToCache(userInfo);
```

Comment:
We may not need this. 
It's done here https://github.com/aiven/klaw/blob/main/core/src/main/java/io/aiven/klaw/service/UsersTeamsControllerService.java#L683

Reply:
Will replace this method with the update metadata missed that we had already